main.js

```diff
@@ -20,6 +20,7 @@ const antiafk = require("mineflayer-antiafk");
 const queueData = require("./queue.json");
 const util = require("./util");
 const save = "./saveid";
+const http = require("http")
```

Comment:
why

Reply:
It's needed to check if 2b2t.io is reachable

Reply:
It also want called before

Reply:
```suggestion
```